plugins/cilium-cni/chaining/generic-veth/generic-veth.go

```diff
@@ -107,6 +107,15 @@ func (f *GenericVethChainer) Add(ctx context.Context, pluginCtx chainingapi.Plug
 			}
 
 			vethIP = addrs[0].IPNet.IP.String()
+
+			addrsv6, err := netlink.AddrList(link, netlink.FAMILY_V6)
+			if err == nil && len(addrsv6) > 0 {
```

Comment:
Do we also want to log an error if there's no error and no IPv6 address? As you said, the interface might just not have any IPv6 address.

Reply:
A bit confused, is that what we already have in the `else` block below?

Reply:
Sorry, I should have been clearer. I don't understand why we are logging an error when there are no IPv6 address and no errors.

Reply:
I think it is logged as a warning? IMHO, it is a bit hard to tell, when there's no ipv6 at all, that whether it is intended or not, that's why I chose to always log a warning msg, but I'm open to suggestions as how we can improve this :) 

If I understand you correctly, are you suggesting we ignore "no ipv6" case and only log if there's a `netlink` error?

Reply:
> If I understand you correctly, are you suggesting we ignore "no ipv6" case and only log if there's a `netlink` error?

Yes. Otherwise, we will log lots of messages for environments with IPv6 disabled.

Reply:
Makes sense, done. PTAL :)